spec/views/application/_navigation.html.haml_spec.rb

```diff
@@ -82,6 +81,23 @@
           .to match %r{<a href=\"\/hundforetag\/#{cmpy_id}\/redigera\">#{text}}
       end
     end
+
+    context 'logged-in menu' do
+      it 'renders logged-in greeting' do
+        text = t('hello', name: member.first_name)
+        expect(rendered).to match /text/
```

Comment:
**You need to interpolate `text` in all of your expected matches  by wrapping it: `#{text}` because without it, you are potentially getting false positives:**

```
it 'renders logged-in greeting' do
         text = t('hello', name: member.first_name)
         expect(rendered).to match /text/
end
```

might return a false positive. 

Ex: This will pass as a false positive:

```
it 'renders logged-in greeting' do
         text = 'Blorfy McBlorf-Face'
         expect(rendered).to match /text/
end
```

but this will correctly fail:
```
it 'renders logged-in greeting' do
         text = 'Blorfy McBlorf-Face'
         expect(rendered).to match /#{text}/
end
```

(And at the same time you can shut houndci-bot up by putting in some parentheses: `expect(rendered).to match(/#{text}/)` )